public/battle.js

```diff
@@ -1,21 +1,27 @@
-async function startBattle() {
-    const response =  fetch('/api/import-songs/songs', {
-        method: 'GET',
-        headers: {
-          'Content-Type': 'application/json'
-        }
-      })
-    //   .then(response =>{
+const startBattle = () => 
```

Comment:
Fetch wasn't necessary since calls happen when pages loads. just asked page to reload, when logged in. volia!